index.html

```diff
@@ -0,0 +1,2620 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>Newsweek - News, Analysis, Politics, Business, Technology</title>
+
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+      integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.1/css/all.css"
+      integrity="sha384-vp86vTRFVJgpjF9jiIGPEEqYqlDwgyBgEF109VFjmqGmIY/Y4HV4d3Gp2irVfcrp"
+      crossorigin="anonymous"
+    />
+
+    <link rel="stylesheet" type="text/css" href="assets/styles.css" />
+
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="https://g.newsweek.com/themes/newsweek/favicons/favicon-32x32.png"
+    />
+  </head>
+
+  <body class="bg-light">
+    <div class="body-wrapper">
+      <header>
+        <div class="container-fluid px-0 font-head">
+          <!-- NEWSWEEK - HEADER red section -->
+          <div class="header d-flex justify-content-between header-wrapper">
+            <div class="p-2 mt-5 d-none d-lg-inline-block align-items-center">
+              <span>Wed, Jan 20, 2021</span>
+            </div>
+            <div>
+              <a href="https://www.newsweek.com" title="Newsweek Homepage">
+                <svg
+                  class="logo-head pt-4"
+                  version="1.1"
+                  xmlns="http://www.w3.org/2000/svg"
+                  xmlns:xlink="http://www.w3.org/1999/xlink"
+                  x="0px"
+                  y="0px"
+                  viewBox="0 0 396 60"
+                  style="enable-background: new 0 0 395.6 60"
+                  xml:space="preserve"
+                >
+                  <g>
+                    <path
```

Comment:
- [ ] [OPTIONAL] Consider saving your SVG images in an .image_name.svg file and then just calling it's path with the image tag in the HTML.
That will make your HTML file easier to read through and smaller, plus your repo more organized!

Reply:
noted.